Hannah-Sten/TeXiFy-IDEA
# Formatter verbatim

Finds a workaround for/fixes #1194 

#### Changes and additions
- Fix verbatim formatting in some (not all, this is definitely buggy) cases (fixes #1194)
- Disable the formatter on files that start and end with a verbatim environment
- Create inspection on the verbatim environment, with two available quickfixes
  - Move verbatim to new file (on which the formatter is disabled, this will also unbreak the parser in the main file)
  - Surround the verbatim environment with `@formatter:off` and `@formatter:on` commands

#### To test this pull request
- Include a verbatim environment in your document
- Check if there is an inspection that that might break TeXiFy
- Try both quick fixes

#### Wiki
- [x] Updated the wiki: https://github.com/Hannah-Sten/TeXiFy-IDEA/wiki/Writing-organisation-recommendations#verbatim-environments-might-break-texify
